agent/command/git.go

```diff
@@ -45,6 +45,9 @@ const (
 	cloneMethodAccessToken = "access-token"
 
 	generatedTokenKey = "EVERGREEN_GENERATED_GITHUB_TOKEN"
+
+	githubMergeQueueInvalidRefError         = "couldn't find remote ref gh-readonly-queue"
+	githubMergeQueueInvalidRefExternalError = "the GitHub merge SHA is not available most likely because the merge completed or was aborted"
```

Comment:
Not sold on these names, please lmk what you think!

Reply:
lets not call this an external error because I assumed that this would be the one from github because that would make it external 

I think it would be fine to not use a variable and just write the error out in the function since its only used here (and I assume there arent future plans to use it more)

Reply:
Sounds good to me, I'll move it

Reply:
ah wait, sorry if I was unclear but I only meant we should delete the githubMergeQueueInvalidRefExternalError and not the githubMergeQueueInvalidRefError.